vendor/engines/sanger_sequencing/app/models/sanger_sequencing/submission.rb

```diff
@@ -22,9 +22,15 @@ class Submission < ActiveRecord::Base
     alias purchased_at ordered_at
 
     scope :purchased, -> { joins(:order).merge(Order.purchased.order(ordered_at: :desc)) }
-    scope :ready_for_batch, -> { purchased.merge(OrderDetail.new_or_inprocess).where(batch_id: nil) }
     scope :for_facility, ->(facility) { where(orders: { facility_id: facility.id }) }
 
+    BATCHABLE_STATES = %w(new inprocess complete).freeze
```

Comment:
Okay. I think it may be worth checking with the client whether that ticket covers (or could be made to cover) a case where a batch was actually run, but the samples may need to be run again (i.e. if the data was bad). I'm not sure if that's actually a situation they have but that was my impression.